aspnetcore/security/authentication/2fa.md

```diff
@@ -21,7 +21,7 @@ uid: security/authentication/2fa
 
 By [Rick Anderson](https://twitter.com/RickAndMSFT)
```

Comment:
@Swiss-Devs 
I think you're better off creating a new project using Visual Studio 2017 RC4  and replace the old project.